Introduction.md

```diff
@@ -0,0 +1,75 @@
+
+### Oracle
+
+An oracle is an entity signing messages in reaction to real-world events.
+Before event occurrence the oracle advertises a public nonce that will be used to later produce a signature.
+The public key used for signing must also be known in advance, but need to be unique for each event.
+Once the event has occurred they release a signed message over the event outcome using the previously advertised nonce.
+
+An important feature of DLC is that the oracle does not have the ability to learn about the existence of the contract.
+This characteristic is the reason for the "discreet log" part of the name, as they don't leave any trace visible to the oracle.
+
+### Trustless execution
+
+A second characteristic of DLCs is that both parties involved in the contract do not need to trust each others.
+The setup and execution of the contract is such that they are guaranteed to always be able to unilaterally close the contract.
+In case of a misbehaving counter-party, trying to close a contract on an outcome for which an oracle signature is not known, the honest protocol participant can claim all the funds locked in the contract.
+Parties are also protected against faulty oracles that would not produce a signature at event occurrence through a refund mechanism
+
+While able to unilaterally close the contracts, parties are expected to generally close the contract cooperatively, as it requires one less transaction to be broadcast.
+
+There is however no protection against a party colluding with an oracle.
+
+## Glossary
+
+### Closing Transaction
+
+A transaction spending from the first output of a [CET](#contract-execution-transaction-cet) unlocking the funds using the appropriate [oracle](#oracle) signature and sending them to an address fully controlled by the broadcasting party.
+It should usually be broadcast together with the corresponding CET.
+
+### Contract Execution Transaction (CET)
+
+A transaction spending from the [fund transaction](#fund-transaction) whose outputs represent a possible outcome of a DLC contract.
+The first output can be spent either by one of the contracting party with knowledge of an oracle signature over the outcome, or by their counter-party after a given timeout.
+The second output can be spent directly.
+
+### Fund transaction
+
+An irreversible on-chain transaction locking the collaterals of both contracting parties in an output that can only be spent by mutual consent.
+It can also contain change outputs in the case where the amount input by a party is greater that the desired collateral.
+
+### Mutual Close
+
+A cooperative closing of a DLC using a [mutual closing transaction](#mutual-closing-transaction).
+
+### Mutual Closing Transaction
+
+A transaction spending from the [fund transaction](#fund-transaction) paying to each party the amount that was agreed based on the published event outcome.
+
+### Oracle
+
+Entity publishing signatures over the outcome of real-world events.
```

Comment:
I added a bit longer description let me know what you think. Regarding composability, we might cover that if assuming a single oracle (same public key) since unless I am mistaken it should be the same as event decomposition (e.g. using exponent and mantissa for representing an event outcome)

Reply:
Good to me!